parts/linux/cloud-init/artifacts/secure-tls-bootstrap.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Runs the secure TLS bootstrapping client binary to generate a kubelet client credential
+ConditionPathExists=/opt/azure/tlsbootstrap/tls-bootstrap-client
+Wants=network-online.target
+After=network-online.target
```

Comment:
Should this have a `Before=kubelet.service`?

Reply:
From reading docs it seems `Before=` is strictly for startup/shutdown ordering, so I think to answer my own question, it should be fine?

https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Before=

Reply:
added per offline discussion